client/lib/url/test/omit-url-params.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * Internal dependencies
+ */
+import omitUrlParams from '../omit-url-params';
+
+describe( 'omitUrlParams()', () => {
+	describe( 'when no URL is supplied', () => {
+		test( 'should return null if the string is not a URL', () => {
+			const actual = omitUrlParams();
+			const expected = null;
+			expect( actual ).toEqual( expected );
+		} );
+	} );
+
+	describe( 'when a URL is supplied', () => {
+		describe( 'when no omitting params are supplied', () => {
+			test( 'should return the URL without modification', () => {
+				const url = 'http://example.com/path?query=banana&query2=pineapple&query3=avocado';
+				const actual = omitUrlParams( url );
+				const expected = 'http://example.com/path?query=banana&query2=pineapple&query3=avocado';
+				expect( actual ).toEqual( expected );
```

Comment:
For primitives, I think we should prefer strict equality `toBe`:

```suggestion
				expect( actual ).toBe( expected );
```

Reply:
👍